src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -8376,6 +8332,8 @@ namespace WaterThermalTanks {
         } else if (Tank.SourceOutletStratNode > 0) {
             // otherwise set it to the temperature of the source outlet node
             Tank.SourceOutletTemp = Tank.Node(Tank.SourceOutletStratNode).TempAvg;
+            // Output the average inlet temperature for the timestep
+            Tank.SourceInletTemp = SourceInletTempSum / SecInTimeStep;
```

Comment:
Is this possibly the source of the small diffs in 5ZoneVAV-ChilledWaterStorage-Stratified?

Reply:
Why is the source side inlet node temperature being set here? Doesn't the tank take the temperature at the source inlet node and use that? If there were a pipe between the source inlet node and the outlet node of the attached component, would the temperature on either side of the pipe match?